tests/test_pragma_parser.py

```diff
@@ -0,0 +1,90 @@
+import pytest
```

Comment:
I'm not sure if we run `black` over the `tests` directory but it seems like this file could be formatted, especially the imports.

Reply:
Done! I did it thanks to `black` and `isort`.